client/my-sites/marketing/do-it-for-me/site-info-collection.tsx

```diff
@@ -1,72 +1,76 @@
 import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 import ActionPanel from 'calypso/components/action-panel';
 import ActionPanelBody from 'calypso/components/action-panel/body';
-import ActionPanelFooter from 'calypso/components/action-panel/footer';
+import CardHeading from 'calypso/components/card-heading';
 import FormattedHeader from 'calypso/components/formatted-header';
-import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import Spinner from 'calypso/components/spinner';
 import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
-import { getProductsList } from 'calypso/state/products-list/selectors';
+import { getCurrentUserName } from 'calypso/state/current-user/selectors';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
-const TemporaryTypeformContainer = styled.div`
-	width: 95%;
-	height: 18em;
-	padding: 1em 0 0 1em;
-	background-color: #eee;
-	font-size: 2em;
-	color: #b8b8b8;
+function getTypeformId(): string {
+	return config< string >( 'difm_typeform_id' );
+}
+
+const LoadingContainer = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 500px;
+	justify-content: center;
 `;
 
-function SiteInformationCollection() {
+const SiteInformationCollection = React.memo( () => {
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
-	const [ isAddingToBasket, setIsAddingToBasket ] = useState( false );
-	const products = useSelector( getProductsList );
-
-	const { replaceProductsInCart } = useShoppingCart();
+	const username = useSelector( getCurrentUserName );
+	const [ isFormSubmitted, setIsFormSubmitted ] = useState( false );
+	const translate = useTranslate();
 
-	const onAddDIFMProductsToCart = async () => {
-		setIsAddingToBasket( true );
-		const difmProduct = fillInSingleCartItemAttributes(
-			{ product_slug: 'wp_difm_lite' },
-			products
-		);
-		const premiumPlan = fillInSingleCartItemAttributes(
-			{ product_slug: 'value_bundle' },
-			products
-		);
-		await replaceProductsInCart( [ difmProduct, premiumPlan ] );
-		page( `/marketing/do-it-for-me/site-info/${ selectedSiteSlug }` );
-	};
 	const onSubmit = async () => {
 		// After Submitting typeform checkout
-		await onAddDIFMProductsToCart();
-		page( `/checkout/${ selectedSiteSlug }` );
+		setIsFormSubmitted( true );
+		page( `/checkout/${ selectedSiteSlug }/wp_difm_lite` );
 	};
 
 	return (
 		<ActionPanel>
 			<ActionPanelBody>
 				<FormattedHeader
 					brandFont
-					headerText="Tell us more about your site"
-					subHeaderText="We need some basic details to build your site, you will also be able to get a glimpse of what your site will look like"
+					headerText={ translate( 'Tell us more about your site' ) }
+					subHeaderText={ translate(
+						'We need some basic details to build your site, you will also be able to get a glimpse of what your site will look like'
+					) }
```

Comment:
Maybe we should hold off on the translations until we have the content finalized.

Reply:
Good catch!